src/wp-includes/blocks/index.php

```diff
@@ -28,7 +28,7 @@ function register_core_block_style_handles() {
 
 	static $core_blocks_meta;
 	if ( ! $core_blocks_meta ) {
-		$core_blocks_meta = require ABSPATH . WPINC . '/blocks/blocks-json.php';
+		$core_blocks_meta = require BLOCKS_PATH . 'blocks-json.php';
```

Comment:
I think it's better to leave this as is, since you already hard-code the `blocks/` folder below, so let's not use a constant here.

On another note, you could move the definition of `$includes_path` further up in the function and then update this line to
```suggestion
		$core_blocks_meta = require $includes_path . 'blocks/blocks-json.php';
```